viewing_party/party.py

```diff
@@ -1,23 +1,133 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:

```

Comment:
Great work completing you first project. I noticed you were checking the values using `None` as a comparison, you could also simplify it by doing something like this:
```
    if not title or not genre or not rating:
```